packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -192,6 +195,18 @@ export function getInternalReactConstants(
     };
   }
 
+  let StrictModeBits = 0;
+  if (gte(version, '18.0.0-alpha')) {
+    // 18+
+    StrictModeBits = 0b011000;
+  } else if (gte(version, '16.9.0')) {
+    // 16.9 - 17
+    StrictModeBits = 0b1;
+  } else if (gte(version, '16.3.0')) {
+    // 16.3 - 16.8
+    StrictModeBits = 0b10;
+  }
+
```

Comment:
This feels potentially bug prone if someone forgets to update this when they update strict mode bits. Is there a way to make this more robust to changes?

Reply:
Yeah. Unfortunately this is the way the rest of the code in `getInternalReactConstants` is too :(

Fortunately changing the reconciler code will cause `/store-test` to fail to tip us off to this coupling, but it's not ideal.